### PR TITLE
fix(autotracing): honor cpusys sampling interval

### PR DESCRIPTION
**Pull request by zjncs**
## Summary

Make CPU system event tracing honor its configured sampling interval.

## Problem

`newCPUSys` validates and stores `Config.CPUSys.Interval` but registers a hard-coded 20-second scheduler interval, so configuration changes have no effect on sampling cadence.

## Changes

- Return the configured interval in `EventTracingAttr.Interval`.

## Verification

- Base SHA: `$(git rev-parse main)`.
- Current main inspection confirmed `intervalSeconds` is used for tracer state while `Interval` was fixed at 20.
- `git diff --check` passed.
- Go tests unavailable because this environment has no `go` or `gofmt`; CI should run the project suite.

## Risk

Low; default behavior remains unchanged when the default interval is used.

AI-assisted contribution; implementation and verification performed against current main.

### CODE REVIEW

File: core/autotracing/cpusys.go [hunk 1]
```diff
@@ -67,7 +67,7 @@ func newCPUSys() (*tracing.EventTracingAttr, error) {
 			perfDuration:     time.Duration(perfDurationSeconds) * time.Second,
 			threshold:        threshold,
 		},
-		Interval: 20,
+		Interval: int(intervalSeconds),
```
Comment (huhong-web) [inline core/autotracing/cpusys.go:70]: [P2] [CR-001] No test proves a non-default configured interval reaches EventTracingAttr.Interval. Add a unit regression for this registration contract.\n\nFix: add the focused regression described above.

**Review by huhong-web — COMMENTED** [review]
## Code Review

One non-blocking finding is included inline: core/autotracing/cpusys.go:70.

## Architecture & Design
- Verdict: Reasonable with a focused verification improvement.
- Evidence: core/autotracing/cpusys.go:70 changes a behavioral contract without proving the stated edge case.

## Scope & Focus
- Verdict: Focused
- Primary objective: Honor the configured sampling interval.
- Necessary changes: core/autotracing/cpusys.go:70 and a focused regression test.
- Out-of-scope changes: None.
- Split plan: Not required.

## Review Convergence
- Mode: full
- Base SHA: ae0badc0a1fed0a1403ff352edd33723aeafaf3e
- Reviewed HEAD: 0f8a39faf5b79068eacad839880f28348c254c61
- Previous reviewed HEAD: none
- Scope verdict: Focused
- Coverage: Changed files and direct configuration/test contracts; two independent review perspectives.
- Prior findings: 0 open / 0 fixed / 0 verified / 0 withdrawn
- New findings: 1 FIRST_PASS / NON-BLOCKING
- Late non-blocking findings: None
- Full-review reset reason: Not required
- Completeness declaration: At HEAD 0f8a39faf5b79068eacad839880f28348c254c61, all changed human-authored files and assigned dimensions were covered; all known blockers are included.

## Verification
- Build/Lint/Integration: NOT RUN — prohibited by review no-build policy.
- Targeted verification: Not required.

---
*Co-authored by: gpt-5.6-terra*
